src/app/routine-home/routine-home.page.html

```diff
@@ -55,8 +44,13 @@ <h5>
     </ion-item>
   </div>
 
+    <h4 class="text-center text-primary-default">
+      My Self Care Goal
+    </h4>
+
   <ion-text color="#B7472A">
-    <h5>My Self Care Goal</h5>
+    <h5 class="text-center mt-2 ml-5 mr-5 mb-10">I would like to achieve work/life balance and feel more energetic daily.
```

Comment:
I don't think this line is supposed to be hardcoded. You add this through the Routine Edit page at the bottom.